src/main/resources/META-INF/plugin.xml

```diff
@@ -94,19 +94,9 @@
   <change-notes><![CDATA[
     <h2>New Features</h2>
       <ul>
-        <li>Updated file sync message from "Syncing Files" to "CodeSync: Syncing Files..."</li>
-        <li>Added marking of git commit hash along with repo init.</li>
-        <li>Added background S3 file upload capability for single new files.</li>
-        <li>Added the ability for users to reconnect account.</li>
-        <li>Async file upload to speed up repo setup.</li>
-        <li>Handling for disabled accounts and the ability to enable it again.</li>
-        <li>Integration with Github pull request to watch pull request playback.</li>
-    <h2>Bug Fixes</h2>
-      <ul>
-        <li>Fixed an error in handle buffer. Added checks for null values.</li>
-        <li>Updated messaging for repo disconnect confirmation prompt.</li>
-        <li>Only process diffs of the authenticated user.</li>
-        <li>Disabled config file reset logic in case of yaml error.</li>
+        <li>Added new error message "Only 1 private repository is allowed in Free Plan"</li>
+        <li>Added custom error codes class: CustomErrorCodes</li>
```

Comment:
This is shown to the end user, so we should be descriptive about it, should not talk much about technical details

Reply:
Also, we can keep previous lines for next couple releases since it highlights recent CHANGELOG